packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -46,35 +40,37 @@ export interface ComboBoxNextProps<T>
    */
   PortalProps?: UseComboBoxPortalProps;
   /**
-   * If `true`, the component will be disabled.
+   * Controlled prop. Controls the value for the Combo Box input.
+   */
+  inputValue?: string;
+  /**
+   * Controlled prop. Controls the Highlighted item in the Combo Box list.
    */
-  disabled: boolean;
-  /* Highlighted index for when the list is controlled. */
   highlightedItem?: string;
-  /* Selected value for when the list is controlled. */
+  /**
+   * Controlled prop. Controls the Selected value in the Combo Box list.
+   */
   selected?: string;
-  /* Initial selected value for when the list is controlled. */
-  defaultSelected?: string;
   /**
-   * The source of combobox items.
+   * Initial input value for when the list is uncontrolled.
    */
-  source: T[];
+  defaultValue?: string;
   /**
-   * Callback for blur event
+   * Initial selected value for when the list is uncontrolled.
```

Comment:
not sure if it'd be beneficial  to include in docs that defaultSelected takes precedence over defaultValue when both are present?

Reply:
will follow up with docs update once content has reviewed, @sion-docs fyi.